redhat-operators/devconsole/devconsole-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,190 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
```

Comment:
Please add required annotations as defined in our docs: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md